test/unit/jsTest.js

```diff
@@ -0,0 +1,234 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const cleanupTestApp = require('../util/cleanupTestApp')
+const generateTestApp = require('../util/generateTestApp')
+const klawSync = require('klaw-sync')
+const fork = require('child_process').fork
+
+describe('JavaScript Section Test', function () {
+  const appDir = path.join(__dirname, '../app/jsTest')
+  const test1 = 'var a = 7'
+  const test2 = 'var b = "turkey"'
+  const test3 = 'var c = true'
+  let staticDirname = 'js'
+  let pathsOfStaticJS = [
+    path.join(appDir, 'statics', 'js', 'a.js'),
+    path.join(appDir, 'statics', 'js', 'b.js'),
+    path.join(appDir, 'statics', 'js', 'c.js')
+  ]
+  let pathsOfCompiledJS = [
+    path.join(appDir, 'statics', '.build', 'js', 'a.js'),
+    path.join(appDir, 'statics', '.build', 'js', 'b.js'),
+    path.join(appDir, 'statics', '.build', 'js', 'c.js')
+  ]
+  let staticJSFiles = [
+    test1,
+    test2,
+    test3
+  ]
+
+  // function to compile the static folder and files
+  let generateStaticFolder = () => {
```

Comment:
Switch this to a function declaration (i.e. `function generateStaticsFolder () {`